uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/my-leave/app.js

```diff
@@ -8,7 +8,7 @@ define([
   'common/services/angular-date/date-format',
   'leave-absences/shared/ui-router',
   'leave-absences/my-leave/modules/config',
-  'leave-absences/my-leave/components/my-leave',
+  'leave-absences/my-leave/components/my-leave-container',
```

Comment:
Got this error 'reqangular.min.js?ooqxmb:91 Uncaught Error: Mismatched anonymous define() module: function...' when debugging is turned on. 
Forgot to document, doing it.